index.js

```diff
@@ -331,12 +332,15 @@ wss.on('connection', async function connection (client) {
   })
 })
 
-server.listen(port, config.hostname, (error) => {
-  if (error) {
-
-  }
-  logger.info(`HTTP Server listening on ${config.hostname} port ${port}`)
-})
+// only listen once the DB is sync'd
+db.sync().then( () => {
```

Comment:
Async/await is preferred in new code over promises, would prefer if it was written to use this.